com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Bob.java

```diff
@@ -418,6 +422,13 @@ private static CommandLine parse(String[] args) {
             helpFormatter.printHelp("bob [options] [commands]", options);
             System.exit(0);
         }
+        if (cmd.hasOption("ce") || cmd.hasOption("pk")) {
+            System.out.println("Android signing using certificate and private key is no longer supported. You must use keystore signing.");
+            HelpFormatter helpFormatter = new HelpFormatter( );
+            helpFormatter.printHelp("bob [options] [commands]", options);
+            System.exit(1);
```

Comment:
Above it says "deprecated", but here it's an error?
To me, "deprecated" means that it's available, but will go away soon.

Reply:
That's true. Should we perhaps keep the old way of generating an APK and use that if cert and key is provided (and emit a warning that we will remove it)?